README.md

```diff
@@ -198,7 +198,7 @@ found in `doc_deps.deb.txt`:
 
 	sudo apt-get install `cat doc_deps.deb.txt`
 
-### Building the Documentation
+### Documentation
```

Comment:
I'd either leave it unchanged or change it to something meaningful like `Build Instructions`.

Reply:
Sure! I'll do changes!
Thanks a bunch!